test/messages.ts

```diff
@@ -472,7 +472,7 @@ describe('fragmentation', function() {
         container = rhea.create_container();
         listener = container.listen({port:0, max_frame_size:16384} as any);
         listener.on('listening', function() {
-            sender = container.connect({port:(listener.address() as AddressInfo).port, max_frame_size:16384}).attach_sender();
+            sender = container.connect({port:(listener.address() as any).port, max_frame_size:16384}).attach_sender();
```

Comment:
I think this is fine. I had to make this change to satisfy TS in the previous PR. Casting it to any in the tests should be  fine.